### PR TITLE
typography component to encapsulate behavior

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

File: src/v2/components/UI/Typography/index.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import cn from 'classnames';
+
+import useStyles from './styles';
+
+type TypographyProps = {
+  type?: 'caption',
+  className?: string,
+};
+
+const Typography = ({type = 'text', className, ...props}: TypographyProps) => {
```
Comment (sunnygleason) [inline src/v2/components/UI/Typography/index.jsx]: This name conflicts with the material-ui/core `Typography` component - can we use an alternate name?

Comment (manuel-calavera) [reply src/v2/components/UI/Typography/index.jsx]: @sunnygleason why is it problem at all? linter or build work fine

**Comment by sunnygleason** [comment]
@manuel-calavera looks good!

* a slight rename would be great because the material-ui `Typography` component is frequently used in practice.

File: src/v2/components/UI/Typography/index.jsx (hunk 1 above)
Comment (manuel-calavera) [reply src/v2/components/UI/Typography/index.jsx]: Gave it ```CTypography``` name meaning custom

**Pull request merged**